shared/src/business/entities/cases/CalendaredCase.ts

```diff
@@ -0,0 +1,93 @@
+import { DOCKET_NUMBER_SUFFIXES } from '../EntityConstants';
+import { DocketEntry } from '@shared/business/entities/DocketEntry';
+import { IrsPractitioner } from '../IrsPractitioner';
+import { JoiValidationConstants } from '../JoiValidationConstants';
+import { JoiValidationEntity } from '../JoiValidationEntity';
+import { PrivatePractitioner } from '../PrivatePractitioner';
+import joi from 'joi';
+
+export class CalendaredCase extends JoiValidationEntity {
+  public docketEntries?: DocketEntry[];
```

Comment:
👏 👏  👏 for only returning what is needed here. 

Does this need to be an entity? Can the endpoint that returns this define a stricter return type in this shape instead?

Reply:
Forgive me if I'm misunderstanding: is the alternative to define a stricter return type and manually strip out the superfluous data?

Reply:
After much discussion, we're leaving the CalendaredCase entity in place. I will respond to this comment with another brach including possible refactoring